lib/cmd/fh3/services/list.js

```diff
@@ -16,13 +16,10 @@ module.exports = {
   'describe' : {
     'json' : i18n._('Output into json format')
```

Comment:
I assumed that this was added in PR. 
What I meant is that we can set this flag to be available globally.